<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_main_window</class>
 <widget class="QMainWindow" name="frm_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>10</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le_callsign">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_itu_zone">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cq_zone">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_dxcc_number">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_dxcc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_mode">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_band">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_year">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_callsign">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Callsign</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_band">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Band</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_year">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_dxcc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DXCC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_mode">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_itu_zone">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ITU Zone</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_dxcc_number">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DXCC Number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cq_zone">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>127</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CQ Zone</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search_qsl">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="grv_logo">
    <property name="geometry">
     <rect>
      <x>865</x>
      <y>20</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tbl_search_view">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>1131</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>690</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_download">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>690</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_unmark_all">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>690</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Unmark all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mark_all">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>690</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mark all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit_program">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>690</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuUpload">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionSingle_Card_Import"/>
    <addaction name="actionBulk_Card_Import"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionVersionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuUpload"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSingle_Card_Import">
   <property name="text">
    <string>Single Card Import</string>
   </property>
  </action>
  <action name="actionBulk_Upload">
   <property name="text">
    <string>Bulk Upload</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionVersionInfo">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBulk_Card_Import">
   <property name="text">
    <string>Bulk Card Import</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
